src/Kunstmaan/MediaBundle/Helper/MediaManager.php

```diff
@@ -160,4 +219,26 @@ public function getFolderAddActions()
 
         return $result;
     }
+
+    /**
+     * @throws Exception
+     */
+    public function isExtensionAllowed(File $file): bool
+    {
+        $mimeType = $this->mimeTypes->guessMimeType($file->getRealPath());
+        $extension = $this->mimeTypes->getExtensions($mimeType)[0];
+        if (!in_array($extension, self::ALLOWED_EXTENSIONS, true)) {
+            return false;
+        }
+
+        if (in_array($extension, self::IMAGE_EXTENSIONS, true)) {
```

Comment:
I would return early here to avoid deeper nesting of the if's

```php
if (!in_array($extension, self::IMAGE_EXTENSIONS, true)) {
    return true;
}
```